docs/reference/errors-and-warnings/NU5131.md

```diff
@@ -43,7 +43,7 @@ The package author wants to prevent package consumers from writing code that dir
 </references>
 ```
 
-This works for projects using the package with `packages.config`, but to work as intended with `ProjectReference`, the project must also contain the following files:
+This works for projects using the package with `packages.config`, but to work as intended with `PackageReference`, the package must also contain the following files:
```

Comment:
What do you think of something like this?

```suggestion
This package will not work as intended when using `PackageReference`. To fix this, the package must also contain the following files:
```

Reply:
Is there a link that explains what `PackageReference` is? If so, please add links